lib/util/wrap.js

```diff
@@ -7,6 +7,8 @@
  * @param {number} [opts.width] Maximum characters per line including the margin
  */
 module.exports = (msg, opts = {}) => {
+  if (parseInt(opts.margin) === 0) return msg || '';
```

Comment:
With this line, I'm addressing another issue: The amount of spaces applied as a margin is always at least one, even if you specify it to be zero.
Unfortunately, I couldn't figure out why this happens.